### PR TITLE
qt5: add activeqt module

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **qt/5.*.***

fixes conan-io/conan-center-index#7441

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by SSE4 — DISMISSED** [review]

**Comment by ohanar** [comment]
So ActiveQt puts its headers put under `include/ActiveQt` and not `include/QtAx*`, so AxBase should probably add that include directory.

File: recipes/qt/5.x.x/conanfile.py [hunk 1]
```diff
@@ -1132,6 +1136,15 @@ def _create_plugin(pluginname, libname, type, requires):
 
         if self.options.qtxmlpatterns:
              _create_module("XmlPatterns", ["Network"])
+        
+        if self.options.qtactiveqt:
+            _create_module("AxBase", ["Gui", "Widgets"])
```
Comment (ohanar) [inline recipes/qt/5.x.x/conanfile.py:1141]:
```suggestion
            _create_module("AxBase", ["Gui", "Widgets"])
            self.cpp_info.components["qtAxBase"].includedirs = ["include", os.path.join("include", "ActiveQt")]
```

This worked for me.

**Review by madebr — APPROVED** [review]

File: recipes/qt/5.x.x/conanfile.py (hunk 1 above)
Comment (prince-chrismc) [reply recipes/qt/5.x.x/conanfile.py:1141]: https://doc.qt.io/qt-5/qaxbase.html

> Header:  #include \<QAxBase\>

Comment (ericLemanissier) [reply recipes/qt/5.x.x/conanfile.py:1141]: Indeed https://github.com/qt/qtactiveqt/blob/5.15.2/src/activeqt/axshared.prx
Thanks!

Comment (ericLemanissier) [reply recipes/qt/5.x.x/conanfile.py:1141]: @prince-chrismc the documentation is fine, it's just that QAxBase (and others) include file is not in the same folder as usual.

Comment (prince-chrismc) [reply recipes/qt/5.x.x/conanfile.py:1141]: Reading documentation was never my strong suite :) Perfect we trust our Qt expert ❤️

**Review by prince-chrismc — APPROVED** [review]

**Review by madebr — APPROVED** [review]

File: recipes/qt/5.x.x/conanfile.py (hunk 1 above)
Comment (uilianries) [reply recipes/qt/5.x.x/conanfile.py:1141]: > Perfect we trust our Qt expert heart

I can't agree more! :D

**Review by uilianries — APPROVED** [review]

**Review by ohanar — APPROVED** [review]

**Pull request merged**